voice/src/main/kotlin/EncryptionMode.kt

```diff
@@ -1,28 +1,15 @@
-package dev.kord.voice
-
-import dev.kord.common.annotation.KordVoice
-import kotlinx.serialization.SerialName
-import kotlinx.serialization.Serializable
-
-@KordVoice
-@Serializable
-public enum class EncryptionMode {
-    @SerialName("xsalsa20_poly1305")
-    XSalsa20Poly1305,
-
-    @SerialName("xsalsa20_poly1305_suffix")
-    XSalsa20Poly1305Suffix,
+@file:GenerateKordEnum(
+    name = "EncryptionMode", valueType = STRING,
+    docUrl = "https://discord.com/developers/docs/resources/auto-moderation#auto-moderation-rule-object-trigger-types",
```

Comment:
this link is wrong, if there is none, just leave it out

Reply:
huh weird. i remember changing that from when i copied it. guess i undid it somehow.

Reply:
Could you also regenerate the result? It still has the wrong link.